File: learn_py_with_js/class/circle.py
import math


class Circle:
    pi = math.pi
    def __init__(self, radius):
        self.radius = radius

    @classmethod
    def create_circle(cls, radius):
        return Circle(radius)

    def get_area(self):
        return self.pi * self.radius ** 2

    def get_circumference(self):
        return 2 * self.pi * self.radius

    def __str__(self):
        return f'Circle with radius {self.radius}'


circle = Circle(5)
print(circle)
print(circle.get_area())
print(circle.get_circumference())

File: learn_py_with_js/class/circle.js
class Circle {

    static pi = Math.PI;

    constructor(radius) {
        this.radius = radius;
    }

    static createCircle(radius) {
        return new Circle(radius);
    }

    gerArea() {
        return Circle.pi * this.radius ** 2;
    }

    getCircumference() {
        return 2 * Circle.pi * this.radius;
    }

    toString() {
        return `Circle with radius ${this.radius}`;
    }
}

const circle = Circle.createCircle(10);
const circleArea = circle.gerArea();
const circleCircumference = circle.getCircumference();
const circleString = circle.toString();

console.log(circleArea);
console.log(circleCircumference);
console.log(circleString);